.vs/config/applicationhost.config

```diff
@@ -0,0 +1,1054 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think the `.vs` directory is supposed to be source controlled. Please remove it and add `.vs` to `.gitignore`?